### ISSUE TITLE
Recorded streams lose color

### ISSUE THREAD

**Issue by thejihuijin** [IntelRealSense/realsense_sdk#22]
I'm trying to use the record/playback code with the Intel Realsense R200 on ubuntu 16.04. 

I added glfw windows to view the stream in the playback code.
When I record more than 250 frames, the images start getting darker/losing color as seen below:
![image](https://cloud.githubusercontent.com/assets/4277830/24984188/d2526f4a-1fa0-11e7-80c3-c90d03bf92db.png)

Original/Beginning:
![image](https://cloud.githubusercontent.com/assets/4277830/24984199/e7a448fa-1fa0-11e7-8f19-1c46b093ed4c.png)

Why do the colors fade? This is not an issue I face when streaming from the camera directly.
Any advice/help would be greatly appreciated. 

thanks

**Comment by zivsha** [comment]
Hi,

Is this happening only when you are using playback? or are you seeing the same thing during record?

**Comment by thejihuijin** [comment]
Hi just to provide a little more information:

It happens during record when I add the GLFW windows as well.
I've recently received the SR300 camera and have tested the code with that camera. It also sees a fade to grey issue towards the end of the recording.

EDIT:
The SR300 glfw window retained color throughout the recording session but the playback was grey.

**Comment by thejihuijin** [comment]
I ran a few more tests and suspect the issue has to do more with my machine's specs than the code itself so I closed the issue. 

I saved the images to file during playback and they were fine, so it was probably my graphics card failing to render it properly.

On a different note, I don't know if this was intended or not but when using the feature rs::record::device->pause_record(), it still "writes" blank frames to file.
